### PR TITLE
feat: return geo info in boot

### PR DESCRIPTION
**Pull request by idoshamun**

### CODE REVIEW

File: .infra/index.ts [hunk 1]
```diff
@@ -306,6 +306,9 @@ if (isAdhocEnv) {
         { targetPort: 3000, port: 80, name: 'http' },
         { targetPort: 9464, port: 9464, name: 'metrics' },
       ],
+      backendConfig: {
+        customRequestHeaders: ['X-Client-Region:{client_region}'],
+      },
```
Comment (capJavert) [inline .infra/index.ts:311]: Is `x-client-region` blocked from req headers, eg. clients can't inject it from req headers?

File: src/routes/boot.ts [hunk 2]
```diff
@@ -130,6 +136,13 @@ type BootMiddleware = (
   res: FastifyReply,
 ) => Promise<Record<string, unknown>>;
 
+const geoSection = (req: FastifyRequest): BaseBoot['geo'] => {
+  return {
+    region: req.headers['x-client-region'] as string,
+    ip: req.ip,
```
Comment (capJavert) [inline src/routes/boot.ts:142]: Do we need the ip to be returned publicly? Even with boot being a private endpoint it opens the attack vector which can dox the client's IP.

**Review by capJavert — COMMENTED** [review]
Nice feature 👌

File: src/routes/boot.ts (hunk 2 above)
Comment (idoshamun) [reply src/routes/boot.ts:142]: Originally, I planned to provide the IP to paddle for the localization but decided to go with a more privacy-oriented way once I found this geolocation trick. We can remove it and there will be no breaking changes

File: .infra/index.ts (hunk 1 above)
Comment (idoshamun) [reply .infra/index.ts:311]: haven't tested it but i think gcp lb overrides it no matter what. we can try

File: src/routes/boot.ts (hunk 2 above)
Comment (idoshamun) [reply src/routes/boot.ts:142]: https://github.com/dailydotdev/daily-api/pull/2465

**Pull request merged**